src/templates/ausfcfg.conf.j2

```diff
@@ -0,0 +1,17 @@
+configuration:
+  groupId: {{ ausf_group_id }}
+  nrfUri: {{ nrf_url }}
+  sbi:
+    bindingIPv4: 0.0.0.0
+    port: {{ sbi_port }}
+    registerIPv4: {{ ausf_url }}
```

Comment:
Reviewing [this PR](https://github.com/canonical/sdcore-udm-operator/pull/4) I started wondering if it wasn't "more correct" to use an IP address here instead of the URL. I know there's a DNS that handles that, but the param name says `IPv4` after all :) 
WDYT?

Reply:
I switched to IP as discussed in the daily.